core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -184,6 +185,13 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
           error("whitelist must be specified when using new consumer in mirror maker.")
           System.exit(1)
         }
+
+        if (!Utils.loadProps(options.valueOf(consumerConfigOpt)).keySet().contains(NewConsumerConfig.PARTITION_ASSIGNMENT_STRATEGY_CONFIG))
+          println("WARNING: The default partition assignment strategy of the new-consumer-based mirror maker will " +
```

Comment:
Maybe we should print to stderr instead? This is what we do in other tools for deprecation warnings (see e.g. ConsoleConsumer).